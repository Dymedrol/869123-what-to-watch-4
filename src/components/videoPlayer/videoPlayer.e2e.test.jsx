import React from 'react';
import Enzyme, {mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import VideoPlayer from './videoPlayer.jsx';

const movie = {
  title: `Fantastic Beasts: The Crimes of Grindelwald`,
  src: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
};

Enzyme.configure({
  adapter: new Adapter(),
});

describe(`Тест состояний плеера`, () => {
  it(`Состояние Play`, () => {
    const player = mount(
        <VideoPlayer
          isPlaying = {true}
          cardData = {movie}
          isMuted = {true}
        />
    );

    expect(player.props().isPlaying).toEqual(true);
  });

  it(`Состояние stop`, () => {
    const player = mount(
        <VideoPlayer
          isPlaying = {false}
          cardData = {movie}
          isMuted = {true}
        />
    );

    expect(player.props().isPlaying).toEqual(false);
  });
});
