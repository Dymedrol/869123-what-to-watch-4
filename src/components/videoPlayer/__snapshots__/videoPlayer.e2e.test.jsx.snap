// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer e2e test 1`] = `
<video
  className="player__video"
  muted={true}
  poster="fantastic-beasts-the-crimes-of-grindelwald.jpg"
/>
`;
